### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by mubaidr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -184,20 +184,20 @@
...
   "devDependencies": {
-    "@types/mocha": "^2.2.44",
-    "@types/node": "^7.0.48",
-    "@types/resolve": "0.0.6",
-    "cross-env": "^5.1.1",
-    "mocha": "^3.5.3",
-    "typescript": "^2.6.2",
-    "vscode": "^1.1.10"
+    "@types/mocha": "^5.2.0",
+    "@types/node": "^10.1.4",
```
Comment (CiGit) [inline package.json]: VSCode runs on node 7.9.0

File: package.json [hunk 2]
```diff
@@ -184,20 +184,20 @@
...
+    "cross-env": "^5.1.6",
+    "mocha": "^5.2.0",
+    "typescript": "^2.8.3",
+    "vscode": "^1.1.18"
   },
   "dependencies": {
-    "ignore": "^3.3.7",
-    "prettier": "1.12.1",
+    "ignore": "^3.3.8",
+    "prettier": "1.13.2",
```
Comment (CiGit) [inline package.json]: update again :smile: and I suspect a new version soon.

File: package.json (hunk 1 above)
Comment (mubaidr) [reply package.json]: Done!

Comment (CiGit) [reply package.json]: Help -> About

Comment (mubaidr) [reply package.json]: Done!

**Comment by CiGit** [comment]
What does this update bring, except prettier, I've not read any patch note / Changelog.

By the way could you add an entry to CHANGELOG.md ? A minor prettier bump deserves it ! with a link to their blog post.

**Comment by mubaidr** [comment]
All dependencies including `prettier` have been updated to latest (& compatible) version. 

Updated change-log too. 👍

**Comment by azz** [comment]
1.13.4 was pushed yesterday.

**Comment by azz** [comment]
Missed the changelog :)

**Comment by mubaidr** [comment]
Will update the change-log. 🔢 

BTW the tests fail on Windows. (travis-ci is fine) 🤔

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,134 +6,138 @@ All notable changes to the "prettier-vscode" extension will be documented in thi
 
 ## [Unreleased]
 
+## [1.3.4]
```
Comment (CiGit) [inline CHANGELOG.md]: It certainly won't be version 1.3.4

Let it under [unreleased], a script does the rest :smiley:

**Pull request merged**